src/test/testing-library/UiTooltipWrapper.test.tsx

```
@@ -1,21 +1,20 @@
+const tooltipContent = 'Tooltip Text';
+const tooltipRole = 'tooltip';
 
 jest.mock('@mui/material', () => ({
   ...jest.requireActual('@mui/material'),
   useMediaQuery: jest.fn(),
 }));
 
 describe('WrapperUiTooltip', () => {
-  const setup: (props?: UiTooltipProps) => void = () =>
+  const setup = (props?: UiTooltipProps) =>
     render(<WrapperUiTooltip title={tooltipContent}>{triggerText}</WrapperUiTooltip>);
```

ESLint

[error] Expected setup to have a type annotation.

(@typescript-eslint/typedef)